### PR TITLE
API version review

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR modifies some bluetooth classes in order to use api version 2.2.0 instead of 3.0.0.
The versions in the code is updated accordingly.

### CODE REVIEW

**Comment by MMaiero** [comment]
@pierantoniomerlino I have made a couple of changes. Please have a review

File: kura/examples/org.eclipse.kura.example.ble.tisensortag.dbus/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -6,7 +6,7 @@ Bundle-Version: 1.0.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Service-Component: OSGI-INF/*.xml
 Import-Package: org.eclipse.kura;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```
Comment (MMaiero) [inline kura/examples/org.eclipse.kura.example.ble.tisensortag.dbus/META-INF/MANIFEST.MF]: Is it ok to have [1.0,2.0)? I mean, the examples use the apis that were added for example with revision 1.3 or not?

File: kura/examples/org.eclipse.kura.example.eddystone.advertiser/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -7,8 +7,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Service-Component: OSGI-INF/*.xml
 Import-Package: org.eclipse.kura;version="[1.4,2.0)",
  org.eclipse.kura.ble.eddystone;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
- org.eclipse.kura.bluetooth.le.beacon;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```
Comment (MMaiero) [inline kura/examples/org.eclipse.kura.example.eddystone.advertiser/META-INF/MANIFEST.MF:10]: Do the same changes need to be applied here?

File: kura/examples/org.eclipse.kura.example.eddystone.scanner/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -9,8 +9,8 @@ Bundle-ActivationPolicy: lazy
 Bundle-ClassPath: .
 Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.ble.eddystone;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
- org.eclipse.kura.bluetooth.le.beacon;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```
Comment (MMaiero) [inline kura/examples/org.eclipse.kura.example.eddystone.scanner/META-INF/MANIFEST.MF:12]: Do the same changes need to be applied here?

File: kura/examples/org.eclipse.kura.example.ibeacon.advertiser/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -7,8 +7,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Service-Component: OSGI-INF/*.xml
 Import-Package: org.eclipse.kura;version="1.4.0",
  org.eclipse.kura.ble.ibeacon;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
- org.eclipse.kura.bluetooth.le.beacon;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```
Comment (MMaiero) [inline kura/examples/org.eclipse.kura.example.ibeacon.advertiser/META-INF/MANIFEST.MF:10]: Do the same changes need to be applied here?

File: kura/examples/org.eclipse.kura.example.ibeacon.scanner/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -9,8 +9,8 @@ Bundle-ActivationPolicy: lazy
 Bundle-ClassPath: .
 Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.ble.ibeacon;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
- org.eclipse.kura.bluetooth.le.beacon;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```
Comment (MMaiero) [inline kura/examples/org.eclipse.kura.example.ibeacon.scanner/META-INF/MANIFEST.MF:12]: Do the same changes need to be applied here?

File: kura/examples/org.eclipse.kura.example.eddystone.advertiser/META-INF/MANIFEST.MF (hunk 2 above)
Comment (pierantoniomerlino) [reply kura/examples/org.eclipse.kura.example.eddystone.advertiser/META-INF/MANIFEST.MF:10]: I'm so sure about that... The original commit for the BT doesn't touch these example. They are compatible also with the old version.

File: kura/test/org.eclipse.kura.internal.ble.test/src/test/java/org/eclipse/kura/internal/ble/util/BluetoothLeUtilTest.java [hunk 6]
```diff
@@ -0,0 +1,155 @@
+package org.eclipse.kura.internal.ble.util;
```
Comment (MMaiero) [inline kura/test/org.eclipse.kura.internal.ble.test/src/test/java/org/eclipse/kura/internal/ble/util/BluetoothLeUtilTest.java:13]: Copyright header missing

**Review by MMaiero — APPROVED** [review]

**Pull request merged**